map_org_names.yaml

```diff
@@ -1621,7 +1621,7 @@ mappings:
   - ['^[[:space:]]*joyent[[:space:]]*$','Joyent, Inc.']
   - ['^[[:space:]]*欢聚时代[[:space:]]*$','Joyy Inc.']
   - ['^[[:space:]]*japan[[:space:]]perl[[:space:]]association[[:space:]]*$','JPA']
-  - ['^[[:space:]]*(ipmorganchase|jp[[:space:]]morgan[[:space:]]chase|jpmorgan|jpmorgan[[:space:]]*chase[[:space:]]*&?[[:space:]]*co\\.?|jpmorgan[[:space:]]chase[[:space:]]&[[:space:]]co\\.?)[[:space:]]*$','JPMorgan Chase']
+  - ['^[[:space:]]*(jp[[:space:]]morgan|ipmorganchase|jp[[:space:]]morgan[[:space:]]chase|jpmorgan|jpmorgan[[:space:]]*chase[[:space:]]*&?[[:space:]]*co\\.?|jpmorgan[[:space:]]chase[[:space:]]&[[:space:]]co\\.?)[[:space:]]*$','JPMorgan Chase']
```

Comment:
I remember also seeing name "IP Morgan" - I know this is a typo, but we should include that in the mapping IMHO.